src/components/molecules/timeline/timeline.stories.tsx

```diff
@@ -40,6 +40,38 @@ EventTimelineWithLinks.args = {
   ],
   listDescription: 'This is the event timeline with a line',
 };
+export const EventTimelineWithDescriptions = Template.bind({});
+EventTimelineWithDescriptions.args = {
+  events: [
+    {
+      name: 'event 1',
+      date: '1999-01-01',
+      link: {
+        text: 'Go to bioRxiv',
+        url: 'https://preprint.url',
+      },
+    },
+    {
+      name: 'event 2',
+      date: '2000-02-02',
+      link: {
+        text: 'Go to version',
+        url: 'https://version.url',
+      },
+      eventDescription: 'Version description',
```

Comment:
When `eventDescription` is present along with link then the `eventDescription` is used as the aria label.